minecraftServers/Controllers/ServersController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using minecraftServers.Repositories;
+using minecraftServers.Servers;
+using System.Text.RegularExpressions;
+
+namespace minecraftServers.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class ServersController : ControllerBase
+{
+    private readonly IServersRepository _serversRep;
+    private readonly List<ServerList> _servers = new();
+    public ServersController(IServersRepository serverRep)
+    {
+        _serversRep = serverRep;
+    }
+
+    [HttpGet]
+    public ActionResult<List<ServerList>> GetAll()
+    {
+        return Ok(_serversRep.GetAll());
+    }
+
+    [HttpPost("/api/servers/add")]
```

Comment:
You can leave the path blank. There is no need to specify this method with `add` path, because in REST the POST method is usually used for adding stuff